core/js/prototype-nav.js

```diff
@@ -26,32 +24,28 @@ try {
 }
 
 // Set up unique IDs for all module titles
-$prototypeNav
-  .find('.br-tree-dir-title')
-  .each(function () {
-    let moduleIds = $(this)
-      .parentsUntil('.br-bordered-list')
-      .children('.br-tree-dir-title')
-      .map(function () {
-        return $(this).text();
-      })
-      .get();
-
-    // Replace space by -
-    moduleIds = moduleIds.map((moduleId) => {
-      return moduleId.split(" ").join("-")
-    })
-
-    $(this).attr('id', moduleIds.join('-'));
-  });
+moduleLabels.forEach(function (element) {
+  let moduleId = [].slice
+    .call(
+      element.parentElement.querySelectorAll('.br-tree-dir-title')
+    )
+    .map(function (title) {
+      // Replace space by -
+      return title.innerText.split(' ').join('-');
+    });
+
+  element.setAttribute('id', moduleId.join('-'));
+});
```

Comment:
```suggestion
moduleLabels.forEach(function (element) {
  const moduleId = element.innerText.split(' ').join('-');

  element.setAttribute('id', moduleId.join('-'));
});
```

I have the feeling this could also be just this, if there are no sibling titles, is that possible?

Reply:
Hmm yes, could be a big simplification. Not sure why the original code is so convoluted.

Reply:
After looking through the pug a couple times, I'm sure that this simplification still works, there's never any module siblings, and even if there are, they wouldn't have the same id.